[2024-03-23T09:35:32.961+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: source_to_gcs.get_data_from_source manual__2024-03-23T09:35:29.981958+00:00 [queued]>
[2024-03-23T09:35:32.982+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: source_to_gcs.get_data_from_source manual__2024-03-23T09:35:29.981958+00:00 [queued]>
[2024-03-23T09:35:32.983+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-23T09:35:33.013+0000] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): get_data_from_source> on 2024-03-23 09:35:29.981958+00:00
[2024-03-23T09:35:33.025+0000] {standard_task_runner.py:60} INFO - Started process 3609 to run task
[2024-03-23T09:35:33.042+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'source_to_gcs', 'get_data_from_source', 'manual__2024-03-23T09:35:29.981958+00:00', '--job-id', '283', '--raw', '--subdir', 'DAGS_FOLDER/sourcetobucket.py', '--cfg-path', '/tmp/tmp4p77gr8g']
[2024-03-23T09:35:33.048+0000] {standard_task_runner.py:88} INFO - Job 283: Subtask get_data_from_source
[2024-03-23T09:35:33.159+0000] {task_command.py:423} INFO - Running <TaskInstance: source_to_gcs.get_data_from_source manual__2024-03-23T09:35:29.981958+00:00 [running]> on host ceb99a37e23c
[2024-03-23T09:35:33.341+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='source_to_gcs' AIRFLOW_CTX_TASK_ID='get_data_from_source' AIRFLOW_CTX_EXECUTION_DATE='2024-03-23T09:35:29.981958+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-23T09:35:29.981958+00:00'
[2024-03-23T09:35:47.250+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-23T09:35:47.267+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=source_to_gcs, task_id=get_data_from_source, execution_date=20240323T093529, start_date=20240323T093532, end_date=20240323T093547
[2024-03-23T09:35:47.335+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-23T09:35:47.486+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
